tools/infrastructure/api_compare.py

```diff
@@ -0,0 +1,487 @@
+    print "\n" + "---" * 60 + "\n"
+    rpc_color = colors.BOLD + colors.HEADER
+    print rpc_color + rpc_name + colors.ENDC
+    print colors.BOLD + "\nMobile API: " + colors.ENDC
+    print "Parameters quantity: ", len(mob_params)
+    print "Parameters list: ", sorted(mob_params.keys())
+    print colors.BOLD + "HMI API: " + colors.ENDC
+    print "Parameters quantity: ", len(hmi_params)
+    print "Parameters list: ", sorted(hmi_params.keys())
+    print "\n"
+    print("{}Parameters absent in Mobile APIs: {}{}".
+          format(colors.WARN, mob_absent_params, colors.ENDC))
+    print("{}Parameters absent in HMI APIs: {}{}".
+          format(colors.WARN, hmi_absent_params, colors.ENDC))
+    print "\n"
+
+
+def console_print(summary_result):
+    """Function which prints summary result to console"""
+    for rpc_name in sorted(summary_result.keys()):
+        print "\n" + "---" * 60
+        print colors.HEADER + rpc_name + colors.ENDC
+        for problematic_item in summary_result[rpc_name]:
+            item = summary_result[rpc_name][problematic_item]
+            if len(item) > 0:
+                print colors.UNDERLINE + problematic_item + colors.ENDC
+            if type(item) is not dict:
+                print("{}{}{}".format(colors.WARN, item, colors.ENDC))
+            elif type(item) is dict:
+                for param in item.keys():
+                    item_print = colors.UNDERLINE + param + colors.ENDC
+                    print "{} {}".format("Parameter name: ", item_print)
+                    res_val = item[param]
+                    for key in res_val:
+                        print key, ":", colors.FAIL, res_val[key], colors.ENDC
+
+
+def print_summary_info(summary_result, args):
+    """Function which prints summary comparison info(if any).
+    Output type depends on command line args
+    """
+    summary_color = colors.UNDERLINE + colors.BOLD + colors.BLUE
+    print "\n"
+    print summary_color, "SUMMARY COMPARISON RESULT:\n", colors.ENDC
+
+    if len(summary_result) == 0:
+        print colors.BOLD + " === NO PROBLEMS FOUND ===" + colors.ENDC
+        return
+
+    if args.output == "console":
+        console_print(summary_result)
+    if args.output == "json":
+        json_summary_result = dict_to_json(summary_result)
+        print json_summary_result
+    if args.output == "xml":
+        json_summary_result = dict_to_json(summary_result)
+        temp = json.loads(json_summary_result)
+        xml_summary_result = json_to_xml(temp)
+        print xml_summary_result
+
+
+def handle_hmi_absent_params(hmi_absent_params, rpc_name, summary_result):
+    """Function which writes hmi absent parameters(if any)
+    to summary result
+    """
+    if len(hmi_absent_params) > 0:
+        fill_summary_result(
+            rpc_name, "Missed in HMI APIs:",
+            hmi_absent_params,
+            summary_result
+        )
+
+
+def handle_mob_absent_params(mob_absent_params, rpc_name, summary_result):
```

Comment:
@AByzhynar can you combine   handle_hmi_absent_params and handle_mob_absent_params? 

```
def handle_absent_params(absent_params, rpc_name, summary_result, hmi_type):
  if len(absent_params) > 0:
  fill_summary_result(
+            rpc_name, "Missed in {} APIs:".format(hmi_type),
+            hmi_absent_params,
+            summary_result
+        )